src/app/sponsors/page.tsx

```diff
@@ -0,0 +1,77 @@
+import Paragraph from '@/components/Paragraph';
+import Sponsors from '@/components/Sponsors';
+import Title from '@/components/Title';
+import type { SponsorType } from '@/data/sponsors';
+import Image from 'next/image';
+
+const SPONSOR_TYPE_COLORS = {
+    gold: '#FCC018',
+    silver: '#C3C3C3',
+    bronze: '#E8903F',
+} as const satisfies Record<SponsorType, string>;
+
+export default function SponsorsPage() {
+    return (
+        <div className="flex flex-col items-center gap-10">
+            <div className="flex gap-8">
+                <Image
+                    src="/images/yellow-duck.svg"
+                    alt="Duck"
+                    width={500}
+                    height={500}
+                    className="hidden w-20 md:block"
+                />
+                <Title colour="yellow">{`${new Date().getFullYear()} Sponsors`}</Title>
```

Comment:
Not sure if we want the year to automatically change as the sponsors don't roll over. @rayokamoto Thoughts?

Reply:
Yeah, we don't want the years to change manually. Although we might get some of the same sponsors the year after, we should still update this manually